integration/presto/src/test/scala/org/apache/carbondata/presto/util/CarbonDataStoreCreator.scala

```diff
@@ -270,6 +270,7 @@ object CarbonDataStoreCreator {
     CSVInputFormat.setEscapeCharacter(configuration, loadModel.getEscapeChar)
     CSVInputFormat.setHeaderExtractionEnabled(configuration, true)
     CSVInputFormat.setQuoteCharacter(configuration, loadModel.getQuoteChar)
+    CSVInputFormat.setLineSeparator(configuration, loadModel.getLineSeparator)
```

Comment:
If not null, then only set the line separator. Else it may lead to wrong behavior.
You already did like this in `CommonUtil.scala`
Follow the same

Reply:
> If not null, then only set the line separator. Else it may lead to wrong behavior.
> You already did like this in `CommonUtil.scala`
> Follow the same

`public static void setLineSeparator(Configuration configuration, String lineSeparator) {
    if (lineSeparator != null && !lineSeparator.isEmpty()) {
      configuration.set(LINE_SEPARATOR, lineSeparator);
    }
  }`
This is done inside the setLineSeparator.

Reply:
> If not null, then only set the line separator. Else it may lead to wrong behavior.
> You already did like this in `CommonUtil.scala`
> Follow the same

Because I am not familar with presto, so please help to check whether here my modification is correct or not. Thanks.

Reply:
It is not a presto production code. This changes are in test files. Doesn't matter.